src/lib/zf/zf_stackdump.c

```diff
@@ -147,10 +182,23 @@ void zf_stack_dump(struct zf_stack* stack)
   dump_stack(stimpl);
   zf_dump("============================================================\n");
   dump_zockets(stimpl);
+  dump_attributes(stimpl);
 
   dump_alts(stimpl);
 };
 
+void zf_stack_dump_attr(struct zf_stack* stack)
+{
+  zf_dump("============================================================\n");
+  zf_stack_dump_summary(stack);
+
+  auto stimpl = SkewPointer<zf_stack_impl>(ZF_CONTAINER(struct zf_stack_impl,
```

Comment:
Variable declaration should be at top of the function. To match coding standards. Also this means you can remove some of the whitespace (see above function zf_stack_dump as example).